src/app/drive/types/file-types.ts

```diff
@@ -208,4 +208,11 @@ export const fileExtensionPreviewableGroups: fileExtensionsPreviewableDictionary
   [FileExtensionGroup.Default]: [],
 };
 
+export const thumbnailableImageExtension: string[] = [...imageExtensions['jpg'], ...imageExtensions['png'],
+...imageExtensions['bmp'], ...imageExtensions['gif']];
+
+export const thumbnailablePdfExtension: string[] = [...pdfExtensions['pdf']];
```

Comment:
Here you can just leave it as:
`export const thumbnailablePdfExtension: string[] = pdfExtensions['pdf'];`

Since there will never be another group for thumbnailPdfExtensions, since its not generic like images